plugins/extractors/kafka/kafka.go

```diff
@@ -27,7 +29,29 @@ var defaultTopics = map[string]byte{
 
 // Config holds the set of configuration for the kafka extractor
 type Config struct {
-	Broker string `mapstructure:"broker" validate:"required"`
+	Broker string     `mapstructure:"broker" validate:"required"`
+	Auth   AuthConfig `mapstructure:"auth_config"`
+}
+
+type AuthConfig struct {
+	TLS struct {
+		// Whether to use TLS when connecting to the broker
+		// (defaults to false).
+		Enabled bool `mapstructure:"enabled"`
+
+		// controls whether a client verifies the server's certificate chain and host name
+		// defaults to false
+		InsecureSkipVerify bool `mapstructure:"insecure_skip_verify"`
+
+		// certificate file for client authentication
+		CertFile *string `mapstructure:"cert_file"`
```

Comment:
Can we keep these as `string` rather than `*string`? I don't think it is adding any value by keeping it as a pointer to `string`.